### PR TITLE
Add IngestKustoImageInfoCommand support for layers

### PR DESCRIPTION
**Pull request by MichaelSimons**
Related to #729

### CODE REVIEW

File: src/Microsoft.DotNet.ImageBuilder/src/Commands/IngestKustoImageInfoCommand.cs [hunk 1]
```diff
@@ -80,33 +79,65 @@ private string GetImageInfoCsv()
...
+                                    imageInfo.AppendLine(
+                                       FormatImageCsv(destinationTag, platform, image, tag.SyndicatedRepo, timestamp));
                                 }
                             }
                         }
+
+                        for (int i = 0; i < platform.Layers.Count; i++)
+                        {
+                            layerInfo.AppendLine(FormatLayerCsv(
+                                platform.Layers[i], 0, platform.Layers.Count - i, sha, platform, image, repo.Repo, timestamp));
```
Comment (mthalman) [inline src/Microsoft.DotNet.ImageBuilder/src/Commands/IngestKustoImageInfoCommand.cs:103]: Please add a comment stating that using `0` for the size is just a placeholder until that data becomes available.

File: src/Microsoft.DotNet.ImageBuilder/src/Commands/IngestKustoImageInfoCommand.cs [hunk 2]
```diff
@@ -80,33 +79,65 @@ private string GetImageInfoCsv()
...
+            string timestamp) =>
+                $"\"{layerDigest}\",\"{size}\",\"{ordinal}\",{FormatImageCsv(imageDigest, platform, image, repo, timestamp)}";
+
+        private async Task IngestInfoAsync(string info, string table)
+        {
+            using MemoryStream stream = new();
+            using StreamWriter writer = new(stream);
+            writer.Write(info);
+            writer.Flush();
+            stream.Seek(0, SeekOrigin.Begin);
```
Comment (mthalman) [inline src/Microsoft.DotNet.ImageBuilder/src/Commands/IngestKustoImageInfoCommand.cs:134]: Couldn't all these lines go inside the `if` block below?

Comment (MichaelSimons) [reply src/Microsoft.DotNet.ImageBuilder/src/Commands/IngestKustoImageInfoCommand.cs:135]: I prefer to have the _dry run_ code path do as much as possible without actually doing the actual thing.

File: src/Microsoft.DotNet.ImageBuilder/src/Commands/IngestKustoImageInfoCommand.cs (hunk 1 above)
Comment (MichaelSimons) [reply src/Microsoft.DotNet.ImageBuilder/src/Commands/IngestKustoImageInfoCommand.cs:104]: Added note.

**Review by mthalman — APPROVED** [review]

**Comment by MichaelSimons** [comment]
Waiting to merge this until the production Kusto layer table is created.

**Pull request merged**